### PR TITLE
History import of Sendmail

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -603,6 +603,7 @@ addess->address
 addessed->addressed
 addesses->addresses
 addessing->addressing
+addheaer->addheader
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This looks specific to their codebase.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4102,6 +4104,7 @@ cheched->checked
 cheching->checking
 chechk->check
 chechs->checks
+checkalaises->checkaliases
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Again

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -7001,6 +7005,7 @@ defintition->definition
 defintivly->definitively
 defition->definition
 defitions->definitions
+defned->defend
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest defined too

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -7130,12 +7135,14 @@ delink->unlink
 delivared->delivered
 delivative->derivative
 delivatives->derivatives
+delivermode->deliverymode
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Again specific

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -10964,6 +10971,7 @@ gernerators->generators
 gess->guess
 get's->gets
 get;s->gets
+getcanoname->getcanonname
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Definitely specific.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -13676,6 +13688,7 @@ lazer->laser
 lazyness->laziness
 lcoation->location
 lcuase->clause
+ldapx->ldap
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This seems oddly specific

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -14684,6 +14698,7 @@ modfies->modifies
 modfiy->modify
 modfy->modify
 modfying->modifying
+modhdrs->addhdrs
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Again specific

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -15446,6 +15461,7 @@ normnal->normal
 northen->northern
 northereastern->northeastern
 nortmally->normally
+noslink->nowlink
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Again specific

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -20903,6 +20924,7 @@ smooting->smoothing
 smouth->smooth
 smouthness->smoothness
 smove->move
+smtpgreeetingmessage->smtpgreetingmessage
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Again specific

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -22730,6 +22754,7 @@ tkaes->takes
 tkaing->taking
 tlaking->talking
 tmis->this
+tname->tzname
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Again specific

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -24675,6 +24703,7 @@ woraround->workaround
 worarounds->workarounds
 worbench->workbench
 worbenches->workbenches
+worchester->worcester
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Please capitalise the correction

**Comment by Gelma** [comment]
All the changes requested are done. I'm going to push fixes

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -12751,6 +12757,7 @@ initilize->initialize
 initilized->initialized
 initilizes->initializes
 initilizing->initializing
+inititalization->initialization
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Please can you add the BE variant with an `s`

File: codespell_lib/data/dictionary.txt [hunk 13]
```diff
@@ -13969,6 +13977,7 @@ lokal->local
 lokale->locale
 lokaly->locally
 lolal->total
+lonber->longer
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest loner too

File: codespell_lib/data/dictionary.txt [hunk 14]
```diff
@@ -21549,6 +21563,7 @@ strucutred->structured
 strucutres->structures
 strucuture->structure
 struggel->struggle
+strust->trust
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest strut too

File: codespell_lib/data/dictionary.txt (hunk 12 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 13 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 14 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 12 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: You've added the plural @Gelma , but please can we have the BE versions of both, i.e. `inititalisation->initialisation` and `inititalisations>initialisations`

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
Cool! 🚀

**Pull request merged**